tests/test_packages/test_connections/test_p2p_libp2p_client/test_communication.py

```diff
@@ -733,54 +360,26 @@ class TestLibp2pClientEnvelopeOrderSamePeer(BaseTestLibp2pClientSamePeer):
 
     def test_burst_order(self):
         """Test order of envelope burst is guaranteed on receiving end."""
-        addr_1 = self.connection_client_1.address
-        addr_2 = self.connection_client_2.address
 
+        sender = self.connection_client_1.address
+        to = self.connection_client_2.address
         sent_envelopes = [
-            self._make_envelope(addr_1, addr_2, i + 1, i)
-            for i in range(self.NB_ENVELOPES)
+            self.enveloped_default_message(to, sender) for _ in range(self.NB_ENVELOPES)
         ]
+
         for envelope in sent_envelopes:
-            self.multiplexer_client_1.put(envelope)
+            envelope.message = envelope.message_bytes  # encode
+            self.multiplexers[1].put(envelope)
 
         received_envelopes = []
         for _ in range(self.NB_ENVELOPES):
-            envelope = self.multiplexer_client_2.get(block=True, timeout=20)
+            envelope = self.multiplexers[2].get(block=True, timeout=3)
             received_envelopes.append(envelope)
 
         # test no new message is "created"
         with pytest.raises(Empty):
-            self.multiplexer_client_2.get(block=True, timeout=1)
+            self.multiplexers[2].get(block=True, timeout=1)
 
-        assert len(sent_envelopes) == len(
-            received_envelopes
-        ), f"expected number of envelopes {len(sent_envelopes)}, got {len(received_envelopes)}"
+        assert len(sent_envelopes) == len(received_envelopes)
         for expected, actual in zip(sent_envelopes, received_envelopes):
-            assert expected.message == actual.message, (
-                "message content differ; probably a wrong message "
-                "ordering on the receiving end"
-            )
-
-
-@pytest.mark.asyncio
-async def test_nodeclient_pipe_connect():
-    """Test pipe.connect called on NodeClient.connect."""
-    f = asyncio.Future()
-    f.set_result(None)
-    pipe = Mock()
-    pipe.connect.return_value = f
-    node_client = NodeClient(pipe, Mock())
-    await node_client.connect()
-    pipe.connect.assert_called_once()
-
-
-@pytest.mark.asyncio
-async def test_write_acn_error():
-    """Test pipe.connect called on NodeClient.connect."""
-    f = asyncio.Future()
-    f.set_result(None)
-    pipe = Mock()
-    pipe.write.return_value = f
-    node_client = NodeClient(pipe, Mock())
-    await node_client.write_acn_status_error("some message")
-    pipe.write.assert_called_once()
```

Comment:
These are duplicated tests, which is why I deleted them. The other copies reside here

https://github.com/valory-xyz/open-aea/blob/4b52af44801b1e59c1d1c25db7d1de17e24eab7a/tests/test_packages/test_connections/test_p2p_libp2p/test_communication.py#L376-L397